py/label_microservice/models.py

```diff
@@ -0,0 +1,24 @@
+"""The models packages defines wrappers around different models."""
```

Comment:
Consider using [abstract classes](https://julien.danjou.info/guide-python-static-class-abstract-methods/) to define python interfaces (more suggestions below)

```suggestion
"""The models packages defines wrappers around different models."""
from abc import ABC, abstractmethod
```